src/routes/(admin)/entity/[action]/[id]/EntityListView.svelte

```
@@ -0,0 +1,90 @@
+	);
+
+	// handle sorting
+	function handleSort(columnId: string) {
+		const url = new URL(page.url);
+		const newSortOrder = sortBy === columnId && sortOrder === 'asc' ? 'desc' : 'asc';
+
+		url.searchParams.set('sortBy', columnId);
+		url.searchParams.set('sortOrder', newSortOrder);
+		url.searchParams.set('page', '1');
+
+		goto(url.toString(), { replaceState: false });
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)